packages/core/elasticsearch/core-elasticsearch-server-internal/src/version_check/ensure_es_version.test.ts

```diff
@@ -388,4 +388,38 @@ describe('pollEsNodesVersion', () => {
 
     expect(internalClient.nodes.info).toHaveBeenCalledTimes(2);
   });
+
+  it('switch from startup interval to normal interval after first green status', () => {
+    expect.assertions(1);
+
+    // @ts-expect-error we need to return an incompatible type to use the testScheduler here
+    internalClient.nodes.info.mockReturnValueOnce([createNodes('6.3.0')]);
```

Comment:
NIT I see that other tests use `nodeInfosSuccessOnce` to wrap this. Can't it be used here?

Reply:
Not for marble testing, unfortunately (can't use promises with the test scheduler). Tests using `nodeInfosSuccessOnce` are the non-marble ones.

Reply:
I can create a dedicated marble-testing helper though, would avoid the `@ts-expect-error` on every line (will have it only once in the helper). Will do that!